src/python/pants/base/hash_utils.py

```diff
@@ -4,10 +4,14 @@
 import hashlib
 import json
 import logging
+import typing
```

Comment:
The preferred style is `from typing import X`. Think Typeshed (official type hints repo) established this in their style guide.

Reply:
Yes, but there is a conflict for `Iterable` with `collections.abc` in this same file so I chose `typing.Iterable` instead of from `typing import a, b, c, Iterable as Bob, d, e`.